### PR TITLE
Feat/pre set data controller

### PR DESCRIPTION
**Pull request by isadorabk**
Controllers for get measures and get ingredients type.

### CODE REVIEW

File: controllers/ingredients.type.controller.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    const ingredientsType = await this.Ingredient_type.findAll({
+      attributes: ['id', 'name'],
+    });
+
+    if (ingredientsType) {
+      ctx.body = ingredientsType.map(el => {
+        const res = {
+          ...el.dataValues
+        };
+        return res;
```
Comment (arol) [inline controllers/ingredients.type.controller.js]:
```js
ctx.body = ingredientsType.map(el => el.dataValues);
```

File: routes/routes.js [hunk 2]
```diff
@@ -23,4 +30,10 @@ router
   .post('/ingredients', authMiddleware, ingredientsController.createIngredient)
   .get('/ingredients', authMiddleware, ingredientsController.getIngredients);
 
+// Measure routes
+router.get('/measures', authMiddleware, measuresController.getMeasures);
+
+// Ingredient_type routes
+router.get('/ingredient-type', authMiddleware, ingredientsTypeController.getIngredientsType);
```
Comment (arol) [inline routes/routes.js]: `/ingredient-types`

File: controllers/measures.controller.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    // Check if the method is correct
+    if (ctx.method !== 'GET') throw new Error('Method not allowed');
+
+    // Find all measures
+    const measures = await this.Measure.findAll({
+      attributes: ['id', 'name'],
+    });
+
+    if (measures) {
+      ctx.body = measures.map(el => {
```
Comment (arol) [inline controllers/measures.controller.js]: Same.

**Review by arol — CHANGES_REQUESTED** [review]
Because of the route name, keep consistent, all plural.

**Comment by isadorabk** [comment]
Changes requested implemented :)

**Review by arol — APPROVED** [review]

**Pull request merged**